sdk/azcore/internal/shared/constants.go

```diff
@@ -0,0 +1,34 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package shared
+
+const (
+	ContentTypeAppJSON = "application/json"
+	ContentTypeAppXML  = "application/xml"
+)
+
+const (
+	HeaderAzureAsync        = "Azure-AsyncOperation"
+	HeaderContentLength     = "Content-Length"
+	HeaderContentType       = "Content-Type"
+	HeaderLocation          = "Location"
+	HeaderOperationLocation = "Operation-Location"
+	HeaderRetryAfter        = "Retry-After"
+	HeaderUserAgent         = "User-Agent"
+)
+
+const (
+	DefaultMaxRetries = 3
+)
+
+const (
+	// Module is the name of the calling module used in telemetry data.
+	Module = "azcore"
+
+	// Version is the semantic version (see http://semver.org) of this module.
+	Version = "v0.19.0"
+)
```

Comment:
version.go at the root is a better place for this.  Customers may want to access the package / version information for their logging.

Reply:
We can't place this in the root as it's used across two different packages.